build/yaml/botbuilder-dotnet-ci-twilio-test.yml

```diff
@@ -6,7 +6,7 @@
 name: $(Build.BuildId)
 
 pool:
-  vmImage: $[ coalesce( variables['VMImage'], 'windows-2019' ) ] # or 'windows-latest' or 'vs2017-win2016'
+  vmImage: $[ coalesce( variables['VMImage'], 'windows-2022' ) ] # or 'windows-latest' or 'vs2017-win2016'
```

Comment:
Please drop 'vs2017-win2016' from the comment, as it is no longer supported.

Reply:
Comment updated. Thanks!